src/components/app-map/map-legend.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="map-legend">
+    <img :src="imgUrl" />
+  </div>
+</template>
+
+<script>
+import legendUrl from '@/lib/legend-url'
+
+export default {
+  props: {
+    url: {
+      type: String,
+      required: true
+    },
+    layer: {
+      type: String,
+      required: true
+    }
+  },
+  computed: {
+    imgUrl() {
+      return legendUrl({
+        url: this.url,
+        layer: this.layer
+      });
+    }
+  }
+};
+</script>
+
+<style>
+.map-legend {
+  position: absolute;
+  left: 1rem;
+  bottom: 1rem;
```

Comment:
It cannot go over the mapbox accreditation, now it's a bit on top of it so may be make the the bottom value a bit higher.